package.xml

```diff
@@ -21,13 +23,14 @@
   <url type="bugtracker">https://github.com/ros/urdf_parser_py/issues</url>
   <url type="repository">https://github.com/ros/urdf_parser_py</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
From what I can tell looking elsewhere, with a pure python package we don't need any `buildtool_depend` line, so just remove this.